build/tasks/publish-build-task.coffee

```diff
@@ -103,10 +103,17 @@ getAssets = ->
         assetName = "atom.#{arch}.rpm"
 
       cp sourcePath, path.join(buildDir, assetName)
+      assets = [{assetName, sourcePath}]
 
-      [
-        {assetName, sourcePath}
-      ]
+      # Check for an archive build on a debian build machine.
+      # We could provide a Fedora version if some libraries are not compatible
+      sourcePath = "#{buildDir}/#{appFileName}-#{version}-#{arch}.tar.gz"
```

Comment:
argh, missed that one (mirroring other publishing task, I'll fix the one that are already upstream in a followup PR I guess ;))